covid19-etl/etl/cityofchicago.py

```diff
@@ -333,23 +338,25 @@ def parse_row(self, row):
         column 3 to rest would be for summary_group_demographics for Age group, race , gender and ethincity
         Here we are ignoring records which doesn't have any lab report dates
         """
-        if row[0] != "":
-            row_dict = {}
-            file_latest_date = self.get_unified_date_format(row[0])
-            row_dict[self.headers[0]] = file_latest_date
-            summary_clinical_submitter_id = self.add_summary_clinical(
-                self.convert_str_to_int(row[1]),
-                self.convert_str_to_int(row[2]),
-                self.convert_str_to_int(row[3]),
-                datetime.strptime(file_latest_date, "%Y-%m-%d"),
+        if not row or not row[0]:
+            return
+        record_date = datetime.strptime(row[0], "%Y-%m-%dT%H:%M:%S.%f").strftime(
+            "%Y-%m-%d"
+        )
```

Comment:
oh looks like you could just do `row[0].split("T")[0]` since you only need a string